dandi/tests/test_metadata.py

```diff
@@ -118,95 +100,55 @@ def test_metadata2asset(schema_dir):
-            )
-        ],
-        wasAttributedTo=[
-            Participant(
-                identifier="a1b2c3",
-                source_id=None,
-                strain=None,
-                cellLine=None,
-                vendor=None,
-                age=PropertyValue(
-                    maxValue=None,
-                    minValue=None,
-                    unitCode=None,
-                    unitText="Years from birth",
-                    value="P170DT12212S",
-                    valueReference=None,
-                    propertyID=None,
-                ),
-                sex=SexType(
-                    identifier=AnyUrl(
-                        "http://purl.obolibrary.org/obo/PATO_0000384",
-                        scheme="http",
-                        host="purl.obolibrary.org",
-                        tld="org",
-                        host_type="domain",
-                        path="/obo/PATO_0000384",
-                    ),
-                    name="Male",
-                ),
-                genotype="Typical",
-                species=SpeciesType(
-                    identifier=AnyUrl(
-                        "http://purl.obolibrary.org/obo/NCBITaxon_9606",
-                        scheme="http",
-                        host="purl.obolibrary.org",
-                        tld="org",
-                        host_type="domain",
-                        path="/obo/NCBITaxon_9606",
-                    ),
-                    name="Human",
-                ),
-                disorder=None,
-            )
-        ],
-        contentUrl=None,
-    )
+    json_data = """{
+  "schemaVersion": "1.0.0-rc1",
+  "identifier": "6a42c273881f45e8ad4d538f7ede1437",
+  "keywords": [
+    "test",
+    "sample",
+    "example",
+    "test-case"
+  ],
+  "access": [
+    {
+      "status": "dandi:Open"
+    }
+  ],
+  "repository": "https://dandiarchive.org/",
+  "contentSize": 69105,
+  "encodingFormat": "application/x-nwb",
+  "digest": {
+    "value": "783ad2afe455839e5ab2fa659861f58a423fd17f",
+    "cryptoType": "dandi:sha1"
+  },
+  "wasDerivedFrom": [
+    {
+      "identifier": "tissue03"
+    }
+  ],
+  "wasAttributedTo": [
+    {
+      "identifier": "a1b2c3",
+      "age": {
+        "unitText": "Years from birth",
+        "value": "P170DT12212S"
+      },
+      "sex": {
+        "identifier": "http://purl.obolibrary.org/obo/PATO_0000384",
+        "name": "Male"
+      },
+      "genotype": "Typical",
+      "species": {
+        "identifier": "http://purl.obolibrary.org/obo/NCBITaxon_9606",
+        "name": "Human"
+      }
+    }
+  ]
+}"""
+    data_as_dict = json.loads(json_data)
+    assert data == AssetMeta(**data_as_dict)
     # We need to convert `data` to a `dict` this way instead of with `.dict()`
     # so that enums will be converted to strings.
-    data_as_dict = json.loads(data.json(exclude_unset=True, exclude_none=True))
```

Comment:
Deleting this line assumes that the input JSON is the same as the output from `json.loads(data.json(...))`.  It would probably be better to add an assertion that the two forms are equal (both here and in the other tests).